assets/js/modules/ads/index.js

```diff
@@ -141,5 +143,43 @@ export const registerNotifications = ( notifications ) => {
 
 			return false;
 		},
-	} );
+	},
+	'account-linked-via-google-for-woocommerce': {
+		Component: AccountLinkedViaGoogleForWooCommerceSubtleNotification,
+		priority: 10,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		checkRequirements: ( { select, resolveSelect } ) => {
+			// isWooCommerceActivated, isGoogleForWooCommerceActivated and isGoogleForWooCommerceLinked are all relying
+			// on the data being resolved in getSiteInfo() selector.
+			resolveSelect( CORE_SITE ).getSiteInfo();
```

Comment:
We need to wait until this is resolved:
```suggestion
			await resolveSelect( CORE_SITE ).getSiteInfo();
```

Reply:
wow totally missed to include `await` 🤦  Thanks